mteb/tasks/BitextMining/dan/BornholmskBitextMining.py

```diff
@@ -43,7 +43,13 @@ class BornholmBitextMining(AbsTaskBitextMining):
 """,
         descriptive_stats={
             "n_samples": {"test": 500},
-            "avg_character_length": {"test": 89.7},
+            "avg_character_length": {
+                "test": {
+                    "average_sentence1_length": 49.834,
+                    "average_sentence2_length": 38.888,
+                    "num_samples": 500,
+                }
+            },
```

Comment:
I see what you meant with `avg_character_length`. Sorry about the confusion:

Would probably have it either as Dict[Key, Dict[Split, Value]]:
```
            "n_samples": {"test": 500},
             "average_sentence1_length": {"test": 49.834}...
```

Or Dict[Split, Dict[Key, Value]]:

```
                "test": {
                     "average_sentence1_length": 49.834,
                     "average_sentence2_length": 38.888,
                     "num_samples": 500,
                 }
```
Either is fine by me. The first is closer to the current implementation.

Reply:
I think the second implementation is better because it allows all metrics for a split to be easily viewed, and the code is clearer